server/commands/write_authzmodel.go

```diff
@@ -29,14 +30,18 @@ func NewWriteAuthorizationModelCommand(
 
 // Execute the command using the supplied request.
 func (w *WriteAuthorizationModelCommand) Execute(ctx context.Context, req *openfgapb.WriteAuthorizationModelRequest) (*openfgapb.WriteAuthorizationModelResponse, error) {
-	typeDefinitions := req.GetTypeDefinitions().GetTypeDefinitions()
+	schemaVersion := typesystem.NewSchemaVersion(req.GetSchemaVersion())
+	if schemaVersion == typesystem.SchemaVersionUnspecified {
```

Comment:
Existing clients will not be providing a `schema_version` field in their requests, and thus this will default to `SchemaVersionUnspecified`, which will lead to this error every time.

We need to account for this to make sure we don't break people's existing integrations. What are your thoughts?

Reply:
Nevermind. We return 1.0 on "". Sorry